ops/ansible/roles/bfd-server/tasks/launch.yml

```diff
@@ -2,16 +2,16 @@
 - name: Copy App Server Launcher Script
   template:
     src: bfd-server.sh.j2
-    dest: "{{ data_server_dir }}/bfd-server.sh"
-    owner: "{{ data_server_user }}"
-    group: "{{ data_server_user }}"
+    dest: "{{ ref_dir }}/bfd-server.sh"
+    owner: "{{ user }}"
+    group: "{{ user }}"
     mode: u=rwx,g=rwx,o=rx
   become: true
 
 - name: Create BFD Server Service Definition
   template:
     src: bluebutton-appserver.service.j2
-    dest: "/etc/systemd/system/{{ data_server_appserver_service }}.service"
+    dest: "/etc/systemd/system/bfd-server.service"
```

Comment:
Agreed. I'll replace this (and any other literal `bfd-server` strings) with the `service` variable.

Reply:
Actually, the more I look into this the more I'm less convinced that there should even _be_ a `service` variable. No other Ansible roles use this pattern when installing their systemd services. And, the `service` variable is only being used in once place whereas the string `bfd-server` is used literally throughout the entire BFD Server Ansible role in contexts not really related to the concept of a "service". Also, it would make the Ansible role even more opaque than it already is; an Engineer trying to grok it would now see something like `{{ service}}.sh.j2` instead of `bfd-server.sh.j2` and would need to know _where_ `service` is defined and _what_ `service` is before they understand that it's referring to the startup script and that they should take a look at it.

I'm going to do the _opposite_ and remove `service` entirely and just use `bfd-server` literally throughout the role.

Reply:
Cool, yeah I'm in favor of removing variables whenever they don't actually... vary.